jsx/StaticDataTable.js

```diff
@@ -297,6 +297,20 @@ var StaticDataTable = React.createClass({
       return (searchString.indexOf(searchKey) > -1);
     }
 
+    // Handle array inputs for multiselects
+    if (typeof filterData === 'object') {
+      var match = false;
+      for (let i = 0; i < filterData.length; i += 1) {
+        var searchKey = filterData[i].toLowerCase();
+        var searchString = data.toLowerCase();
+
+        match = (searchString.indexOf(searchKey) > -1);
+        if (match) {
+          return true;
```

Comment:
Perhaps a 'break' is more appropriate here since the 'return match' on line 312 will be returning true under this condition as well - that is if we are trying to avoid multiple return statements. I'm not sure what the convention is on that one.

Reply:
@PapillonMcGill & @HenriRabalais - I followed the convention that had been in place for the existing checks (for number & string inputs). I wouldn't say that the function is returning multiple things, it just returns 'true' if a certain condition is met and if not, it returns the output of another check.

Reply:
@llevitis The multiple return statement will of course return only 1 thing (true or false), but previously, it what considered a bad practice to have multiple return statement inside a function, you would set a variable ($result = true) mid function as needed with possible break statement and a the end, return the variable value.  
Maybe things have change but I still only use 1 return statement per function as it make it clear what is returned, just look at the last line.

Reply:
@llevitis please use `let` or `const` to declare variables that value can or can't change

Also, what you did is just fine (you followed the existing convention to keep coherence in this function code). However, as @HenriRabalais and @PapillonMcGill are saying, it is makes it hard to follow the flow of the function when there is multiple return statement in it. With the exception of some obvious checks/assertions at the start of the function it is preferable that a return variable is declared at the top of the function and its value assignment (optionally) occur later. 
Now, if you don't mind, you could: 1. declare a return value initially set to false. 2. use the `||` operator to get the value of `this.props.Filter`. 3. use a switch statement on `typeof filter.value` then put the comparison code in `case "number":` `case "string"` and `case "object"`. 4. a single return at the bottom